services/src/main/java/org/exoplatform/gamification/connectors/github/rest/GithubWebHookRest.java

```
@@ -0,0 +1,118 @@
+                        receiverId = senderId;
+                        object = infoNode.get("comment").get("_links").get("html").get("href").textValue();
+                        if (senderId != null) {
+                            githubHooksManagement.broadcastGithubEvent(ruleTitle, senderId, receiverId, object);
+                        }
+                    }
+                    break;
+                    case "pull_request_review": {
+                        ruleTitle = "reviewPullRequest";
+                        senderId = githubHooksManagement.getUserByGithubId(infoNode.get("review").get("user").get("login").textValue());
+                        receiverId = senderId;
+                        object = infoNode.get("review").get("html_url").textValue();
+                        if (senderId != null && !infoNode.get("review").get("state").textValue().equals("commented")) {
+                            githubHooksManagement.broadcastGithubEvent(ruleTitle, senderId, receiverId, object);
+                        }
+                        if (infoNode.get("review").get("state").textValue().equals("approved")) {
+                            receiverId = githubHooksManagement.getUserByGithubId(infoNode.get("pull_request").get("user").get("login").textValue());
+                            if (senderId != null && receiverId != null) {
+                                ruleTitle = "pullRequestValidated";
+                                githubHooksManagement.broadcastGithubEvent(ruleTitle, senderId, receiverId, object);
+                            }
+                        }
+                    }
+                    break;
+
+                }
+
+                return Response.ok().build();
+            } catch (Exception e) {
+                return Response.status(Response.Status.INTERNAL_SERVER_ERROR)
+                        .entity("error")
```

This error message is useless. You can instead return the exception message for example.